build now failing
```
======================================================================
ERROR: test_update_status_set_location (corehq.apps.users.tests.fixture_status.TestFixtureStatus)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/users/tests/fixture_status.py", line 62, in test_update_status_set_location
    self.couch_user.set_location(fake_location)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/users/models.py", line 1807, in set_location
    location.group_id
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/jsonobject/utils.py", line 23, in update
    self[key] = value
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/jsonobject/containers.py", line 137, in __setitem__
    wrapped, unwrapped = self.__unwrap(key, value)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/jsonobject/containers.py", line 131, in __unwrap
    return self._wrapper.unwrap(wrapped)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/jsonobject/base_properties.py", line 218, in unwrap
    property_ = self.value_to_property(obj)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/jsonobject/base_properties.py", line 238, in value_to_property
    value, map_types_properties.keys())
BadValueError: value <MagicMock name='mock.group_id' id='140439430674896'> not in allowed types: [<type 'list'>, <type 'int'>, <type 'long'>, <type 'str'>, <type 'datetime.time'>, <type 'unicode'>, <class 'decimal.Decimal'>, <type 'bool'>, <type 'datetime.datetime'>, <type 'float'>, <type 'dict'>, <type 'datetime.date'>, <type 'set'>]
```

--- a/corehq/apps/users/tests/fixture_status.py
+++ b/corehq/apps/users/tests/fixture_status.py
@@ -57,6 +57,7 @@ class TestFixtureStatus(TestCase):
     def test_update_status_set_location(self):
         fake_location = MagicMock()
         fake_location.location_id = "the_depths_of_khazad_dum"
+        fake_location.group_id = "the_depths_of_khazad_dum"
         self.assertEqual(UserFixtureStatus.objects.all().count(), 0)
 
         self.couch_user.set_location(fake_location)
@@ -70,6 +71,7 @@ class TestFixtureStatus(TestCase):
     def test_update_status_unset_location(self):
         fake_location = MagicMock()
         fake_location.location_id = "the_mines_of_moria"
+        fake_location.group_id = "the_mines_of_moria"
         self.couch_user.set_location(fake_location)
         previously_updated_time = UserFixtureStatus.objects.get(user_id=self.couch_user._id).last_modified
 

--- a/corehq/ex-submodules/casexml/apps/case/cleanup.py
+++ b/corehq/ex-submodules/casexml/apps/case/cleanup.py
@@ -74,8 +74,7 @@ def safe_hard_delete(case):
     """
     if not settings.UNIT_TESTING:
         from corehq.apps.commtrack.const import USER_LOCATION_OWNER_MAP_TYPE
-        whitelist_case_types = [USER_LOCATION_OWNER_MAP_TYPE]
-        if case.type not in whitelist_case_types:
+        if case.type != USER_LOCATION_OWNER_MAP_TYPE:
             raise CommCareCaseError("Attempt to hard delete a case whose type isn't white listed")
 
     if case.reverse_indices:

